### DISCUSSION TITLE
Can Babylon native be only used with React Native?

### DISCUSSION THREAD

**Discussion by BraveEvidence** [BabylonJS/BabylonNative#867] [Development]
Is it possible to use Babylon Native with Kotlin for Android, Swift for iOS or with Flutter using dart? Are there any bindings available for it?

**Comment by bghgary** [comment]
_Please use the forum for questions._

Babylon Native is built with C++. You can call into C++ from any of these languages (though I don't know much about Flutter). Babylon Native is built with the intention that you can use it anywhere. We don't have any explicit built-in integrations except for React Native right now.
